src/main/java/Client.java

```diff
@@ -14,5 +23,11 @@ public static void main(String[] args) {
         .build();
 
     // Make the call to the getFlightInfo and GetStream
+    FlightDescriptor flight_desc = FlightDescriptor.command("GetFlightInfo".getBytes(StandardCharsets.UTF_8));
+    final FlightInfo flightInfo = client.getInfo(flight_desc);
+    System.out.println("EndPoints: " + flightInfo.getEndpoints() + "Descriptor: "+ flightInfo.getDescriptor());
+    System.out.println("Ticket: " + flightInfo.getEndpoints().get(0).getTicket());
+    final FlightStream flightStream = client.getStream(flightInfo.getEndpoints().get(0).getTicket());
```

Comment:
Should consume the FlightStream and print data on console

Reply:
fixed